frontend/config-assistant/src/components/toolbar/fetchWebSchemas.ts

```diff
@@ -1,12 +1,12 @@
 import type {SchemaOption} from '@/model/SchemaOption';
 import {JSON_SCHEMA_STORE_CATALOG_URL} from '@/constants';
 
-export async function fetchWebSchemas(): Promise<SchemaOption[]> {
+export async function fetchWebSchemas(toast?: any): Promise<SchemaOption[]> {
```

Comment:
I think you should use the explicit type, not any.
I don't remember what it was, but we use the type in other places. Something like "ToastServiceMethods" is what I remember but I might be wrong

Reply:
@Paul, we tried together that but it did not work, that's why we tried this way, its not new, this already we have in other files, but we did not have this one in this that's what I just added